staging/src/k8s.io/api/networking/v1beta1/types.go

```diff
@@ -178,6 +178,7 @@ type IngressRuleValue struct {
 	// usable by a loadbalancer, like http keep-alive.
 
 	// +optional
+	// HTTP represents detailed http selectors pointing to backends against incoming requests
```

Comment:
"pointing to backends against incoming requests" feels like it might be missing a word?

> HTTP is a list of selectors which are used to match incoming requests to a set of backends

Is this more accurate?

Reply:
updated.